File: WOT-iOS/WOTData/WOTData/model/subclasses/VehicleprofileAmmoPenetration+CoreDataProperties.m
//
//  VehicleprofileAmmoPenetration+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 8/28/18.
//  Copyright © 2018 Pavel Yeshchyk. All rights reserved.
//
//

#import "VehicleprofileAmmoPenetration+CoreDataProperties.h"

@implementation VehicleprofileAmmoPenetration (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileAmmoPenetration *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileAmmoPenetration"];
}

@dynamic avg_value;
@dynamic max_value;
@dynamic min_value;
@dynamic vehicleprofileAmmo;

@end

File: WOT-iOS/WOTApi/source/model/autogenerated/VehicleprofileAmmoPenetration+CoreDataProperties.swift
//
//  VehicleprofileAmmoPenetration+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 5/1/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import CoreData

public extension VehicleprofileAmmoPenetration {
    @nonobjc class func fetchRequest() -> NSFetchRequest<VehicleprofileAmmoPenetration> {
        return NSFetchRequest<VehicleprofileAmmoPenetration>(entityName: "VehicleprofileAmmoPenetration")
    }

    @NSManaged var avg_value: NSDecimalNumber?
    @NSManaged var max_value: NSDecimalNumber?
    @NSManaged var min_value: NSDecimalNumber?
    @NSManaged var vehicleprofileAmmo: VehicleprofileAmmo?
}
